docs/howto/manage-charms.md

```diff
@@ -0,0 +1,207 @@
+(how-to-manage-charms)=
+# How to manage charms
+
+> See first: {external+juju:ref}`Juju | Build a charm <build-a-charm>`, {external+charmcraft:ref}`Charmcraft | Manage charms <manage-charms>`
+
+## Create a repository and initialise it
+
+Create a repository with your source control of choice. Commit the code you have
+added and changed after every significant change, so that you have a record of
+the work, and can revert to an earlier version when required.
+
+```{admonition} Best practice
+:class: hint
+
+Name the repository using the pattern ``<charm name>-operator`` for a single
+charm, or ``<base charm name>-operators`` when the repository will hold
+multiple related charms. For the charm name, see
+{external+charmcraft:ref}`Charmcraft | Specify a name <specify-a-name>`.
+```
+    
+In your new repository, run `charmcraft init` to generate the recommended
+structure for building a charm.
+
+```{note}
+In most cases, you'll want to use `--profile=machine` or `profile=kubernetes`.
+If you are charming an application built with a popular framework, check if
+charmcraft has a {external+charmcraft:ref}`specific profile <tutorial>` for it.
+
+Avoid the default (`--profile=simple`), which provides a demo charm, rather than
+a base for building a charm of your own.
+```
+
+````{tip}
+If your repository will hold multiple charms, or a charm and source for other
+artifacts, like a Rock, create a `charms` folder at the top level, then a folder
+for each charm inside of that one, and run `charmcraft --init` in each charm
+folder. You'll end up with a structure similar to:
+
+```
+my-charm-set-operators/
+├── charms
+│   ├── my-charm
+│   │   ├── charmcraft.yaml
+│   │   ├── pyproject.toml
+│   │   ├── README.md
+│   │   ├── requirements.txt
+│   │   ├── src
+│   │   │   └── charm.py
+│   │   ├── tests
+│   │   │   ├── integration
+│   │   │   │   └── test_charm.py
+│   │   │   └── unit
+│   │   │       └── test_charm.py
+│   │   └── tox.ini
+│   ├── my-charm-dashboard
+|   |   └── ...
+│   └── my-charm-helper
+|   |   └── ...
+├── CONTRIBUTING.md
+├── LICENSE
+├── README.md
+└── rock
+    └── ...
+```
+
+````
+
+## Development tooling
```

Comment:
Try to find an active title. The rule to have all sections in a how-to active, verb-based, has served us well so far, and helped us write tighter docs, so I wouldn't give it up.